scripts/build/with-react.js

```diff
@@ -0,0 +1,13 @@
+import '../../lib/ckeditor/ckeditor';
```

Comment:
Seems like we shouldn't have this line here, because it will pull ckeditor into the `noCkeditor` build, won't it?

Reply:
Right, this shouldn't be there